examples/basic/android/app/src/main/AndroidManifest.xml

```diff
@@ -14,6 +14,7 @@
 
     <application
       android:name=".MainApplication"
+      android:usesCleartextTraffic="true"
```

Comment:
Is there any docs about this?

Reply:
https://github.com/facebook/react-native/issues/22375